arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/timer.kt

```diff
@@ -15,7 +17,7 @@ import kotlinx.coroutines.withTimeoutOrNull
  * }
  * ```
  **/
-@Deprecated("Use delay", ReplaceWith("delay(duration.millis)", "kotlinx.coroutines.delay"))
+@Deprecated("Use delay", ReplaceWith("delay(duration.millis)", "kotlinx.coroutines.delay"), DeprecationLevel.ERROR)
```

Comment:
```suggestion
@Deprecated("Use delay", ReplaceWith("delay(duration.millis)", "kotlinx.coroutines.delay"))
```

Deprecating as `ERROR` breaks the users build, first round of deprecation is always `WARNING`.

Reply:
I will add that for the others, too ! :D